src/test/java/de/qabel/desktop/ui/LayoutControllerTest.java

```diff
@@ -1,15 +1,57 @@
 package de.qabel.desktop.ui;
 
+import de.qabel.core.accounting.QuotaState;
+import de.qabel.core.exceptions.QblInvalidCredentials;
 import javafx.scene.Node;
 import org.junit.Test;
 
+import java.io.IOException;
+
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 public class LayoutControllerTest extends AbstractControllerTest {
 
     public static final int MIN_ITEM_COUNT = 2;
 
+    //same values as the BoxClientStub
+    private QuotaState expectedQuotaState = new QuotaState(24, 100);
+
+    @Test
+    public void testQuotaDescription() {
+        String expected = "668 MB free / 954 MB";
+        long availableQuota = 1000000000L;
+        long usedQuota = 300000000L;
+        String quotaDescription = createController().quotaDescription(usedQuota, availableQuota);
+
+        assertEquals(expected, quotaDescription);
+    }
+
+    @Test
+    public void testRatioByDiff() {
+        int expectedRatio = 30;
+        long availableQuota = 1000000000L;
+        long usedQuota = 300000000L;
+        int ratio = createController().ratioByDiff(usedQuota, availableQuota);
+
+        assertEquals(expectedRatio, ratio);
+    }
+
+    @Test
+    public void testFillQuotaInformation() throws IOException, QblInvalidCredentials {
+        LayoutController controller = createController();
+        QuotaState quota = controller.boxClient.getQuotaState();
+
+        String expectedDesc = controller.quotaDescription(expectedQuotaState.getSize(), expectedQuotaState.getQuota());
+        int expectedRatio = controller.ratioByDiff(expectedQuotaState.getSize(), expectedQuotaState.getQuota());
+
+        assertEquals(expectedQuotaState.getQuota(), quota.getQuota());
+        assertEquals(expectedQuotaState.getSize(), quota.getSize());
+        assertEquals(expectedRatio + "%", controller.quota.getText());
+        assertEquals(expectedRatio, (int) controller.provider.getMinWidth());
+        assertEquals(expectedDesc, controller.quotaDescription.getText());
```

Comment:
Compare your test to this alternative version:
```JAVA
@Test
public void fillsQuotaInformation() throws Exception {
    boxClientStub.setQuotaState(new QuotaState(24, 100));
    createController();
    
    assertEquals("24%", controller.quota.getText());
    assertEquals(24, controller.provider.getMinWidth());
    assertEquals("24 B free / 100 B", controller.quotaDescription.getText());
}
```

Which one is more obvious in that we want to test that the controller initializes those three values? (%, description and width).
Imho using explicit (useful) values is always better than calculating your expectations. Because you should know what you expect in a test.